repos/system_upgrade/el7toel8/actors/migratebrltty/tests/component_test.py

```diff
@@ -0,0 +1,93 @@
+        file_out.write('braille-device bluez:AB-cd:ef:01:23:45\n')
+    current_actor_context.feed(BrlttyMigrationDecision(migrate_file=BRLTTY_CONF, migrate_bt=True, migrate_espeak=False))
+    current_actor_context.run()
+    with open(BRLTTY_CONF, 'r') as file_in:
+        data = file_in.read().strip()
+    _conf_restore(backup)
+    assert data == 'braille-device bluetooth:AB-cd:ef:01:23:45'
+
+def test_actor_migrate_espeak(current_actor_context):
+    backup = _conf_prepare()
+    with open(BRLTTY_CONF, 'w') as file_out:
+        file_out.write('speech-driver es\n')
+    current_actor_context.feed(BrlttyMigrationDecision(migrate_file=BRLTTY_CONF, migrate_bt=False, migrate_espeak=True))
+    current_actor_context.run()
+    with open(BRLTTY_CONF, 'r') as file_in:
+        data = file_in.read().strip()
+    _conf_restore(backup)
+    assert data == 'speech-driver en'
+
+def test_actor_migrate_just_bt(current_actor_context):
+    backup = _conf_prepare()
+    with open(BRLTTY_CONF, 'w') as file_out:
+        file_out.write('braille-device bth:AB-cd:ef:01:23:45\n')
+        file_out.write('braille-device bluez:AB-cd:ef:01:23:45\n')
+        file_out.write('speech-driver es\n')
+    current_actor_context.feed(BrlttyMigrationDecision(migrate_file=BRLTTY_CONF, migrate_bt=True, migrate_espeak=False))
+    current_actor_context.run()
+    with open(BRLTTY_CONF, 'r') as file_in:
+        data = file_in.read().strip()
+    _conf_restore(backup)
+    assert data == 'braille-device bluetooth:AB-cd:ef:01:23:45\nbraille-device bluetooth:AB-cd:ef:01:23:45\nspeech-driver es'
+
+def test_actor_migrate_just_espeak(current_actor_context):
+    backup = _conf_prepare()
+    with open(BRLTTY_CONF, 'w') as file_out:
+        file_out.write('braille-device bth:AB-cd:ef:01:23:45\n')
+        file_out.write('braille-device bluez:AB-cd:ef:01:23:45\n')
+        file_out.write('speech-driver es\n')
+    current_actor_context.feed(BrlttyMigrationDecision(migrate_file=BRLTTY_CONF, migrate_bt=False, migrate_espeak=True))
+    current_actor_context.run()
+    with open(BRLTTY_CONF, 'r') as file_in:
+        data = file_in.read().strip()
+    _conf_restore(backup)
+    assert data == 'braille-device bth:AB-cd:ef:01:23:45\nbraille-device bluez:AB-cd:ef:01:23:45\nspeech-driver en'
+
+def test_actor_migrate_all(current_actor_context):
+    backup = _conf_prepare()
+    with open(BRLTTY_CONF, 'w') as file_out:
+        file_out.write('braille-device bth:AB-cd:ef:01:23:45\n')
+        file_out.write('braille-device bluez:AB-cd:ef:01:23:45\n')
+        file_out.write('speech-driver es\n')
+    current_actor_context.feed(BrlttyMigrationDecision(migrate_file=BRLTTY_CONF, migrate_bt=True, migrate_espeak=True))
+    current_actor_context.run()
+    with open(BRLTTY_CONF, 'r') as file_in:
+        data = file_in.read().strip()
+    _conf_restore(backup)
+    assert data == 'braille-device bluetooth:AB-cd:ef:01:23:45\nbraille-device bluetooth:AB-cd:ef:01:23:45\nspeech-driver en'
```

Comment:
Please use expected, actual order for testing asserts, not visa verse.
`assert 'braille-device bluetooth:AB-cd:ef:01:23:45\nbraille-device bluetooth:AB-cd:ef:01:23:45\nspeech-driver en' == data`